js-src/functions/src/main/scala/org/scalalang/utils/Logger.scala

```diff
@@ -0,0 +1,158 @@
+package org.scalalang.utils
+
+import org.scalajs.dom.console
+
+import scala.reflect.ClassTag
+import scala.scalajs.js
+
+/**
+  * A basic logger for the debugging console
+  */
+object Logger {
```

Comment:
I browsed around for a few scala js logging implementation but didn't find one that fit my needs. I'm open to swapping this out for a different project vs my home-built solution

Reply:
perhaps you could put a comment to that effect at the top of the file, and say briefly what the considerations were

Reply:
You have two actual uses of this logger. I suggest replacing them by a good old `println`.